--- server/src/db_config.rb
+++ server/src/db_config.rb
@@ -14,9 +14,9 @@
   unless $SERVER_NAME
     $SERVER_NAME = "NOT_SV"
   end
-  STORE_TYPE = :sqlite3
+  STORE_TYPE = :mysql
   # memcached server
-  MEMCACHE_CONFIG = 'localhost:11211'
+  MEMCACHE_CONFIG = 'memcached:11211'
   MEMCACHE_OPTIONS = {
     :timeout =>1,
     :namespace => 'unlight'
@@ -30,6 +30,7 @@
     :database =>"unlight_db",
     :encoding => 'utf8',
     :max_connections => 5,
+    :port => 3306,
     :loggers => Logger.new(File.dirname(__FILE__).gsub("src","")+"data/#{$SERVER_NAME}_mysqldb.log", 32, 10*1024*1024)
   }

